@plotly/dash-test-components/src/components/DelayedEventComponent.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
```

Comment:
Adding a set of components with simple behaviors corresponding to various usage scenarios. This component delays `setProps` until later.